app_ex1.js

```diff
@@ -0,0 +1,25 @@
+
+const express = require('express');
+const router = express.Router();
+const dotenv = require("dotenv").config();
+const app = express();
```

Comment:
Please accept this code, so the `.env` file would be able to expand a variable by other variables.

```suggestion
const dotenv = require("dotenv");
const dotenvExpand = require("dotenv-expand");
dotenvExpand.expand(dotenv.config());
```

Take note that you should install the `dotenv-expand` package with `npm i dotenv-expand`, and commit the updated `package.json`.

Then, for example you could use in your `.env`:

```sh
USERNAME=John
HELLO_MESSAGE="Hello ${USERNAME}"
```

Reply:
no problem i would be happy for more info about it :)